_layouts/default.html

```diff
@@ -83,7 +83,7 @@
       <footer class="clearfix">
         <section class="container clearfix">
           <div class="bio">
-            <p>Belfast Ruby is a community of aspiring and experienced developers in Belfast, Northern Ireland. We're passionate about Ruby and Rails, holding <a href="/past-meetups/">monthly meetups</a> to learn and share.</p>
+            <p>Belfast Ruby is a community of aspiring and experienced developers in Belfast, Northern Ireland. We're passionate about Ruby and Rails, holding <a href="/past-meetups/">monthly meetups</a> to learn and share. you would be glad to join the community</p>
```

Comment:
I think the last sentence would read better if it wasn't there. If you delete it I will merge. Thanks very much :D